pubsub/apiv1/pubsubpb/alias.go

```diff
@@ -0,0 +1,801 @@
+)
+
+// Deprecated: Please use vars in: cloud.google.com/go/pubsub/v2/apiv1/pubsubpb
+var (
+	BigQueryConfig_State_name                              = src.BigQueryConfig_State_name
+	BigQueryConfig_State_value                             = src.BigQueryConfig_State_value
+	CloudStorageConfig_State_name                          = src.CloudStorageConfig_State_name
+	CloudStorageConfig_State_value                         = src.CloudStorageConfig_State_value
+	Encoding_name                                          = src.Encoding_name
+	Encoding_value                                         = src.Encoding_value
+	File_google_pubsub_v1_pubsub_proto                     = src.File_google_pubsub_v1_pubsub_proto
+	File_google_pubsub_v1_schema_proto                     = src.File_google_pubsub_v1_schema_proto
+	IngestionDataSourceSettings_AwsKinesis_State_name      = src.IngestionDataSourceSettings_AwsKinesis_State_name
+	IngestionDataSourceSettings_AwsKinesis_State_value     = src.IngestionDataSourceSettings_AwsKinesis_State_value
+	IngestionDataSourceSettings_AwsMsk_State_name          = src.IngestionDataSourceSettings_AwsMsk_State_name
+	IngestionDataSourceSettings_AwsMsk_State_value         = src.IngestionDataSourceSettings_AwsMsk_State_value
+	IngestionDataSourceSettings_AzureEventHubs_State_name  = src.IngestionDataSourceSettings_AzureEventHubs_State_name
+	IngestionDataSourceSettings_AzureEventHubs_State_value = src.IngestionDataSourceSettings_AzureEventHubs_State_value
+	IngestionDataSourceSettings_CloudStorage_State_name    = src.IngestionDataSourceSettings_CloudStorage_State_name
+	IngestionDataSourceSettings_CloudStorage_State_value   = src.IngestionDataSourceSettings_CloudStorage_State_value
+	IngestionDataSourceSettings_ConfluentCloud_State_name  = src.IngestionDataSourceSettings_ConfluentCloud_State_name
+	IngestionDataSourceSettings_ConfluentCloud_State_value = src.IngestionDataSourceSettings_ConfluentCloud_State_value
+	PlatformLogsSettings_Severity_name                     = src.PlatformLogsSettings_Severity_name
+	PlatformLogsSettings_Severity_value                    = src.PlatformLogsSettings_Severity_value
+	SchemaView_name                                        = src.SchemaView_name
+	SchemaView_value                                       = src.SchemaView_value
+	Schema_Type_name                                       = src.Schema_Type_name
+	Schema_Type_value                                      = src.Schema_Type_value
+	Subscription_State_name                                = src.Subscription_State_name
+	Subscription_State_value                               = src.Subscription_State_value
+	Topic_State_name                                       = src.Topic_State_name
+	Topic_State_value                                      = src.Topic_State_value
+)
+
+// Request for the Acknowledge method.
+//
+// Deprecated: Please use types in: cloud.google.com/go/pubsub/v2/apiv1/pubsubpb
```

Comment:
Note: We wanted to strongly push users to our new libs when I wrote this tool. If you are fine with the deprecation warnings on everything that is fine, but just wanted to call out that this will affect godoc for these things. Check out a genproto package as an example.

Reply:
Yeah this is fine. We are trying to move everyone to the new GAPIC and v2 types.